src/Nodes/Order/Order.php

```diff
@@ -54,14 +54,14 @@ public function getOrderDetails($parameters = []): ResponseData
     }
 
     /**
-     * GetOrdersByStatus is the recommended call to use for order management.
+     * Get order list from shopee by time range and order status
      *
-     * @param array|RequestParametersInterface $parameters
+     * @param  array|RequestParametersInterface  $parameters
```

Comment:
Is there any reason two spaces are use instead one space? @param array instead of @param array

Reply:
Use 2 spaces according to this guideline https://laravel.com/docs/8.x/contributions#phpdoc